scripts/upgrade_s3_permissions.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
```

Comment:
This code is quite far from the corresponding Control Panel code dealing with AWS (which I believe is somehow duplication, but I don't mean this is as a bad thing): https://github.com/ministryofjustice/analytics-platform-control-panel/pull/854/files

I wonder if it could be closer to that code and be a django command instead or even migration (probably a bad idea), potentially even reusing some of the code there and/or constants.

You'd have the additional benefit of being part of the same codebase and potentially be easier to discover/modify when AWS change these again (hopefully never and/or in years 😬 )

Just an idea, I know it's not the important part here ☺️

Reply:
Ack, but I was told (for example when I added versioning to S3 buckets) that such migration scripts relating to infrastructure related settings should be in the `ops` repository. It feels to me like this is the right place, and we should not entangle ourselves with Django since this (ultimately) has nothing really to do with AWS related configuration.